lib/Common/Core/SysInfo.cpp

```diff
@@ -397,10 +397,33 @@ LPCWSTR AutoSystemInfo::GetJscriptDllFileName()
     return (LPCWSTR)Data.binaryName;
 }
 
+#ifdef _WIN32
 HMODULE AutoSystemInfo::GetCRTHandle()
 {
-    return GetModuleHandle(_u("msvcrt.dll"));
+    if (Data.crtBase == 0)
+    {
+        Data.crtBase = GetModuleHandle(_u("msvcrt.dll"));
```

Comment:
Fair enough. I was thinking we've already lost if they can corrupt this, but i could see how this might make things a bit harder